release/release_after_kokoro.sh

```diff
@@ -20,6 +20,11 @@ elif [ ! -x "$ECLIPSE_HOME/eclipse" ]; then
     exit 1
 fi
 
+if ! `which -s xml`; then
```

Comment:
I'm not a shell script expert at all, but FYI, I remember this: http://mywiki.wooledge.org/BashFAQ/082.